examples/action/variables.tf

```diff
@@ -48,38 +29,45 @@ variable "resource_group_name" {
   type        = string
 }
 
-variable "provision_namespace" {
-  description = "Provision Namespace"
+##################################################
+# Action Variables
+##################################################
+variable "is_new_action" {
+  description = "Provision Action"
   type        = bool
-  default     = false
+  default     = true
 }
 
-variable "create_package" {
-  description = "Option whether to create a new package"
-  type        = bool
+variable "action_name" {
+  description = "Name of action."
+  type        = string
 }
 
-variable "package_name" {
-  description = "Package Name"
-  type        = string
+variable "action_limits" {
```

Comment:
expand the contents of "action_limits" like it has timeout, memory and log_size

Reply:
Expanded the variables